enclave_wrangler/README.md

```diff
@@ -1,51 +1,52 @@
-## Enclave wrangler
+## **Enclave Wrangler**
 This [documentation can be found here](https://github.com/jhu-bids/TermHub/tree/develop/enclave_wrangler)
 
-## Creating/Updating concept sets via CSV upload
-Example: [sample CSV](https://github.com/jhu-bids/TermHub/blob/develop/test/input/test_enclave_wrangler/test_dataset_upload/type-2-diabetes-mellitus.csv).
+## **Creating/Updating Concept Sets via CSV Upload**
+Example: [Sample CSV](https://github.com/jhu-bids/TermHub/blob/develop/test/input/test_enclave_wrangler/test_dataset_upload/type-2-diabetes-mellitus.csv).
 
-### CSV Schema
-At a minimum, all of these columns not marked 'Optional' in the description are necessary. Additional columns are allowable, but they will be ignored.
+### **CSV Schema**
+At a minimum, all of these columns not marked 'False' in the description are necessary. Additional columns are allowable, but they will be ignored.
```

Comment:
Ahh, a required column. I was thinking about that. After merging, I think I'll change this verbiage to read something like the following instead:
> Non-required columns can can be included in the CSV, but will be ignored during processing.